recipe/meta.yaml

```diff
@@ -21,9 +21,9 @@ requirements:
     - cmake >=3.9
     - ninja-base
   host:
-    - aws-c-common 0.12.4
-    - aws-c-cal 0.9.2
-    - s2n {{ s2n }}  # [linux]
+    - aws-c-common 0.12.6
+    - aws-c-cal 0.9.13
+    - s2n 1.6.2  # [linux]
```

Comment:
All of those packages are in [cbc.yaml](https://github.com/AnacondaRecipes/aggregate/blob/master/conda_build_config.yaml). Should they be?
```
    - aws-c-common {{ aws_c_common }}
    - aws-c-cal {{ aws_c_cal }}
    - s2n  {{ s2n }} # [linux]
```

Will they be compatible with other donstream packages?